src/ucp/rndv/rndv.c

```diff
@@ -1859,7 +1865,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_rtr_handler,
         if (is_pipeline_rndv) {
             status = ucp_rndv_send_start_put_pipeline(sreq, rndv_rtr_hdr);
             if (status != UCS_ERR_UNSUPPORTED) {
-                return status;
+                ucs_debug("pipeline_rndv is failed. fallback to PUT_ZCOPY or AM");
```

Comment:
why need it? e.g. it can be `UCS_ERR_NO_MEMORY` - I guess we should fail operation if faced an unrecoverable error

Reply:
you are right.